app/theme.ts

```diff
@@ -81,6 +81,20 @@ const customTheme = extendTheme({
       },
     },
   },
+  layerStyles: {
+    whiteBg: {
+      bg: "white",
+      color: "brand.text",
+    },
+    darkBg: {
+      bg: "brand.backgroundDark",
+      color: "white",
+    },
+    medBg: {
+      bg: "brand.backgroundMed",
+      color: "white",
+    },
+  },
```

Comment:
I've added this section to easier switch the background color and text color for individual components. You can see this applied in both Footer and Location. When I tried to just reset the global styles within ChakraUI theme, it worked, but it also affected Archaa's value section's background (which we struggled to implement); so I ended up creating the layerStyles instead.